hack/generate_clients.sh

```diff
@@ -4,7 +4,6 @@ set -o nounset
 set -o pipefail
 
 go install k8s.io/code-generator/cmd/client-gen@latest
-go get kubevirt.io/api
```

Comment:
Does this still cause issues now that alexander added this line https://github.com/kubevirt/csi-driver/blob/cc28dcbb0afca0a7cb8a73bc998ab49f864ed560/Makefile#L52  which disables generate during the build step?

Reply:
it's strange to me that we call `go get kubevirt.io/containerized-data-importer-api` a few lines below this but are removing `go get kubevirt.io/api`

Reply:
can we remove this change and follow up on sorting out this generate logic in a separate PR